tests/www/apply/test_process.py

```diff
@@ -3051,7 +3051,8 @@ def test_delete_prior_action(client, snapshot, with_geiq_diagnosis):
             author_kind=AuthorKind.GEIQ,
         )
     # Create transition logs
-    job_application.move_to_prior_to_hire(user=user)
+    with freeze_time("2023-12-12 13:40:00", tz_offset=-1):
```

Comment:
Je me demande si 
```suggestion
    with freeze_time(timezone.now() + timedelta(minutes=3)):
```
ne fonctionnerait pas ?

Reply:
Ça marche et c’est plus simple.